iothub/service/src/JsonSerializerSettingsInitializer.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.Azure.Devices
     /// <summary>
     /// A class to initialize JsonSerializerSettings which can be applied to the project.
     /// </summary>
-    public static class JsonSerializerSettingsInitializer
+    internal static class JsonSerializerSettingsInitializer
```

Comment:
As per our discussion, change the access to internal

Reply:
The access modifiers for the methods down below can be made internal as well.